### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by elvan03**
Created the user profile's screen. it contains information on the user such as name, profile picture and interests. Solves issue #8

### CODE REVIEW

**Comment by GhaliBerbich** [comment]
I linked the Firestore database to the application, specifically the 'profiles' collection with the profile page, so that I can retrieve profile information from the profiles present in the database. I implemented the UserProfileViewModel file as well as the ProfilesRepositoryFirestore file for that. I then used the profileViewModel in the profileScreen to display the informations of the user from the database.

File: app/src/main/java/com/android/sample/model/ProfilesRepositoryFirestore.kt
Comment (aminebenji04) [inline app/src/main/java/com/android/sample/model/ProfilesRepositoryFirestore.kt:1]: LGTM!

File: app/src/main/java/com/android/sample/model/User.kt
Comment (aminebenji04) [inline app/src/main/java/com/android/sample/model/User.kt:1]: Why would a photo be of type String? 
We will maybe add other info to the User data class

File: app/src/main/java/com/android/sample/model/UserProfileViewModel.kt
Comment (aminebenji04) [inline app/src/main/java/com/android/sample/model/UserProfileViewModel.kt:1]: LGTM!

File: app/src/main/java/com/android/sample/ui/profile/Profile.kt
Comment (aminebenji04) [inline app/src/main/java/com/android/sample/ui/profile/Profile.kt:1]: I think that all we have to do is see how it renders on the screen

File: gradle/wrapper/gradle-wrapper.properties
Comment (aminebenji04) [inline gradle/wrapper/gradle-wrapper.properties:1]: Why would we remove this line?

**Review by aminebenji04 — APPROVED** [review]
LGTM ! Just check why you removed the date in gradle/wrapper/gradle-wrapper.properties

**Pull request merged**